.github/workflows/cicd.yml

```diff
@@ -128,11 +143,23 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 15
     needs: [build]
+    environment: prod
```

Comment:
Please introduce environment as a workflow input to open design for extensions in case we would introduce a new environment.
```yml
workflow_dispatch:
  inputs:
    environment:
      description: "Environment"
      type: environment
      required: true
      default: prod
```
You may need to set a default environment in `init` job as we already do with matrix because it is not initialized when workflow is being triggered by push or pull request.